webapp/src/components/studies/StudyCard.tsx

```diff
@@ -380,12 +406,7 @@ export default function StudyCard(props: Props) {
                 <ListItemText>{t("global.export")}</ListItemText>
               </MenuItem>
               {study.managed && (
-                <MenuItem
-                  onClick={() => {
-                    onArchiveClick(study);
-                    handleClose();
-                  }}
-                >
+                <MenuItem onClick={handleArchiveClick}>
```

Comment:
not related to your PR but could you replace the trad of this MenuItem from "global.delete" to "global.archive"